mobile/src/auth/azureAuthProvider.ts

```diff
@@ -225,11 +226,19 @@ export const azureAuthProvider = (): IAsyncAuthProvider => {
       }
 
       if (request.nonce) {
-        if (!validIdToken(request.nonce, result.idToken, request.issuer)) {
+        if (validIdToken(request.nonce, result.idToken, request.issuer)) {
+          const idToken = decode<IdToken>(result.idToken);
+          await SecureStore.setItemAsync("oid", idToken.oid);
```

Comment:
Can we please move these keys we are using (`oid`, `email`, `phoneNumber`) rather to an enum in the types.ts file? I see we are reusing them in other auth folder classes/files and just better for future maintenance to have them in an enum class.

Reply:
Fixed.